mlflow/models/utils.py

```diff
@@ -489,7 +512,15 @@ def _save_example(
     if input_example is None:
         return None
 
-    example = _Example(input_example, no_conversion=no_conversion)
+    # TODO: remove this and all example_no_conversion param after 2.17.0 release
+    if no_conversion is not None:
+        _logger.warning(
+            "The `example_no_conversion` parameter is deprecated since mlflow 2.16.0. "
+            "MLflow no longer converts input examples when logging the model. "
+            "Please remove the parameter from your call args."
```

Comment:
Can we add a message like `This parameter is no longer used and safe to be removed.`?